### ISSUE TITLE
Update that ``C`` can now be accessed via ``from sympy.abc import C``

### ISSUE THREAD

**Issue by gxyd** [sympy/sympy#10440]
Issue #7124 has now been closed, probably solved as mentioned at last comment on the issue.

Hence on [sympy.abc line 15](https://github.com/sympy/sympy/blob/master/sympy/abc.py#L15) the ``C`` can now be removed from it, or from any other place possibly occuring on the same file or any other file.

[Edit]
This is what i get
```
>>> from sympy import *
>>> C
/usr/local/lib/python3.4/dist-packages/IPython/lib/pretty.py:108: SymPyDeprecationWarning: 

C, including its class ClassRegistry, has been deprecated since SymPy
0.7.7. It will be last supported in SymPy version 0.7.7. Use direct
imports from the defining module instead. See
https://github.com/sympy/sympy/issues/7124 for more info.

  return getattr(obj, attr, default)
```

**Comment by gxyd** [comment]
Or this a bug in `DeprecationWarning` message, since `0.7.7` -> `1.0`.

**Comment by asmeurer** [comment]
I will fix all the occurrences of 0.7.7 to 1.0 when I make the release branch. 

I think we should hold off on adding `C` to abc.py until we fully remove `ClassRegistry`.

**Comment by asmeurer** [comment]
That issue was closed prematurely (deprecation removal issues shouldn't be closed until the thing is removed completely, see https://github.com/sympy/sympy/wiki/Deprecating-policy). I'm closing this issue as a duplicate of that one.
